changelog.d/3024.bugfix

```diff
@@ -0,0 +1 @@
+Improved search performance when dealing with cases such as URIs in messages.
```

Comment:
```suggestion
Improve search performance when dealing with cases such as URIs in messages.
```

Feel free to add a `Contributed by @werner291.` to get your name in the changelog :)